hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,154 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Generated UNR file from Synopsys UNR tool with security modules being
+// black-boxed.
+
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Oct  6 16:19:29 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_io_div2_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_io_div4_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_aes_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_hmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_kmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_otbn_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_main_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_io_root_ctrl.u_scanmode_sync
```

Comment:
All these "if (!rst_ni)" are for unused logic in prim_mubi4_sync: https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/prim/rtl/prim_mubi4_sync.sv;drc=10daa25ec9d8bebe6d86b6b20694cad1c2a40448;l=143

I will add explicit coverage exclusions in a separate PR, since I will need to change the mubi_sync templates.

Reply:
sounds good, let me know if you need any help here. 
We could alternatively just remove that unused logic and add lint waivers.  That's also an option.